R/lookup.R

```diff
@@ -18,8 +18,6 @@ prod_nesting_vars_lookup <- c(
 
 calculation_level_lookup <- "company"
 
-end_year_lookup <- 2040
```

Comment:
very tiny nit: there is still test_end_year_lookup <- 2040 in test_financial_functions, this can be removed

Reply:
done